openshift/minio/minio-deployment.yaml

```diff
@@ -0,0 +1,86 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: ${APPLICATION_NAME}-template
+labels:
+  template: ${APPLICATION_NAME}-template
+objects:
+- kind: Deployment
+  metadata:
+    name: ${APPLICATION_NAME}
+    namespace: ${NAMESPACE}
+    labels:
+      app: ${APPLICATION_NAME}
+  spec:
+    replicas: 1
+    template:
+      metadata:
+        name: ${APPLICATION_NAME}
+        labels:
+          app: ${APPLICATION_NAME}
+      spec:
+        volumes:
+          - name: data
+            emptyDir:
+              sizeLimit: 500Mi
+        containers:
+          - name: minio
+            image: quay.io/minio/minio:latest
+            imagePullPolicy: IfNotPresent
+            livenessProbe:
+              httpGet:
+                path: /minio/health/live
+                port: 9000
+              initialDelaySeconds: 120
+              periodSeconds: 20
+            readinessProbe:
+              httpGet:
+                path: /minio/health/ready
+                port: 9000
+              initialDelaySeconds: 120
+              periodSeconds: 20
+            env:
+              - name: MINIO_ROOT_USER
+                value: ${MINIO_ROOT_USER}
+              - name: MINIO_ROOT_PASSWORD
+                value: ${MINIO_ROOT_PASSWORD}
+            resources:
+              requests:
+                memory: 2Gi
```

Comment:
That's a lot of memory. Have you confirmed that it has this heavyweight a footprint?

Reply:
I can tune this backdown NBD, but when I first spun it up, the response to the URL was really slow.

I turned both memory and CPU up from the values that I found were in this template originally and the console seemed to respond a bit more quickly.

Whatever you think is best, I can adjust NBD.